lib/openstax/exchange/client/real_client.rb

```diff
@@ -0,0 +1,80 @@
+require 'oauth2'
+
+module OpenStax
+  module Exchange
+    module Client
+      class RealClient
+        include ClientInstance
```

Comment:
I disagree. See https://github.com/openstax/exchange-ruby/pull/8#discussion_r22813683 for the reasoning.